brasilio_auth/views.py

```diff
@@ -24,3 +27,12 @@ def form_valid(self, *args, **kwargs):
         login(self.request, self.object)
         url = self.request.POST.get('next', None) or self.success_url
         return redirect(url)
+
+
+class ProfileUpdate(UpdateView):
```

Comment:
Como para acessar essa página o usuário precisa estar logado, precisamos adicionar aqui o decorator `login_required` - [veja exemplo na documentação do Django](https://docs.djangoproject.com/en/2.1/topics/class-based-views/intro/#decorating-the-class).